charts/rmd/templates/ingress.yaml

```diff
@@ -0,0 +1,35 @@
+{{- if .Values.ingress.enabled -}}
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: {{ include "rmd.fullname" . }}
+  labels:
+    {{- include "rmd.labels" . | nindent 4 }}
```

Comment:
This is "missing" the component label. There should either be no component labels at all or they should be everywhere.